05_stringer/alextmz/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// IPAddr is below, violets are blue and this is here to shut up golint :-)
```

Comment:
Amusing, but you can also shut up lint by writing a proper doc string :-)

If you really don't want to, you don't need to export the type - golint will stop complaining then too.

Either way, please approach these labs as you would real code, and write the comment as appropriate.

Reply:
Woops, this was not intended to stay there, but I guess I forgot to clean it out before pushing. Done.